### PR TITLE
Update Jetty version

### PR DESCRIPTION
**Pull request by ivanyu**
Closes https://github.com/aiven/http-connector-for-apache-kafka/pull/71
Closes https://github.com/aiven/http-connector-for-apache-kafka/pull/73

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -72,7 +72,7 @@ ext {
     jacksonVersion = "2.13.1" // This Jackson is used in the tests.
     jupiterVersion = "5.8.2"
     kafkaVersion = "2.0.1" // Oldest version supported, new versions are backwards compatible.
-    jettyVersion = "9.4.11.v20180605"
+    jettyVersion = "11.0.8"
```
Comment (reta) [inline build.gradle:75]: @ivanyu  We should be aware that Jetty 11 is Jakarta + JDK-11 baseline while Kafka is still using `javax.*` APIs:

```
+--- org.apache.kafka:connect-runtime:2.0.1
|    +--- org.apache.kafka:connect-api:2.0.1
|    |    +--- org.apache.kafka:kafka-clients:2.0.1 (*)
|    |    +--- org.slf4j:slf4j-api:1.7.25 -> 1.7.36
|    |    \--- javax.ws.rs:javax.ws.rs-api:2.1
```

Probably what you want is to got with Jetty 10.x which is equivalent of Jetty 11 for `javax.*` namespace

**Review by reta — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (ivanyu) [reply build.gradle:75]: Good point, actually tests fail due to some class-not-found with 11.

Comment (reta) [reply build.gradle:75]: Jetty 10 is also JDK-11 baseline

**Comment by ivanyu** [comment]
Ok, it starts looking like a rabbit hole :) It's a test dependency so probably no harm if it's outdated (or - in line with Kafka)

File: build.gradle (hunk 1 above)
Comment (jlprat) [reply build.gradle:75]: But if I'm not mistaken reading the gradle file, this project should be in JDK-11

Comment (reta) [reply build.gradle:75]: Haven't looked but Jetty 9 could be bumped to `9.4.45.v20220203`

**Pull request closed without merging**